src/libs/OptionsListUtils.js

```diff
@@ -343,6 +343,7 @@ function createOption(logins, personalDetails, report, reportActions = {}, {
         result.login = personalDetail.login;
         result.phoneNumber = personalDetail.phoneNumber;
         result.payPalMeAddress = personalDetail.payPalMeAddress;
+        result.alternateText = Str.removeSMSDomain(personalDetail.login);
```

Comment:
This is based on what we do above with the `personalDetail.login`, for example [here](https://github.com/Expensify/App/pull/11023/files#diff-733528d957ae6a80eb863637fa5d54278c89c6121998f3574fa8c0aba935daebR326-R328).

Reply:
Ah, nice catch! Thank you for adding this!